static/src/js/components/DownloadHistory/__tests__/DownloadHistory.test.jsx

```diff
@@ -1,255 +1,226 @@
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
-import Table from 'react-bootstrap/Table'
-import Helmet from 'react-helmet'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { MemoryRouter } from 'react-router-dom'
 
 import * as deployedEnvironment from '../../../../../../sharedUtils/deployedEnvironment'
 import * as AppConfig from '../../../../../../sharedUtils/config'
 
-import Spinner from '../../Spinner/Spinner'
-import PortalLinkContainer from '../../../containers/PortalLinkContainer/PortalLinkContainer'
 import { DownloadHistory } from '../DownloadHistory'
 
-Enzyme.configure({ adapter: new Adapter() })
-
-function setup(props) {
-  const enzymeWrapper = shallow(<DownloadHistory {...props} />)
-
-  return {
-    enzymeWrapper,
-    props
-  }
-}
+jest.mock('../../../../../../sharedUtils/deployedEnvironment', () => ({
+  deployedEnvironment: jest.fn()
+}))
+
+jest.mock('../../../../../../sharedUtils/config', () => ({
+  getEnvironmentConfig: jest.fn()
+}))
+
+jest.mock('../../../containers/PortalLinkContainer/PortalLinkContainer', () => ({
+  __esModule: true,
+  default: ({ children, to, portalId }) => (
+    <a
+      href={`/downloads/${to.pathname.split('/').pop()}${to.search}`}
+      data-testid="portal-link"
```

Comment:
Don't use test ids

Reply:
I'd just mock PortalLinkContainer and test that you are giving it the correct props. That will simplify the test setup quite a bit

```
expect(PortalLinkContainer).toHaveBeenCalledTimes()
expect(PortalLinkContainer).toHaveBeenCalledWith()
```